Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -429,25 +470,39 @@ public override void StartDriver()
                         }
                         break;
 
+                    #endregion
+
+                    #region EDGE
                     case eBrowserType.Edge:
-                        if (Convert.ToInt32(HttpServerTimeOut) > 60)
+                        if (Convert.ToInt32(HttpServerTimeOut) >= 60)
                         {
                             EdgeDriverService service = EdgeDriverService.CreateDefaultService();
+
+                            if (HideConsoleWindow == true)
+                                service.HideCommandPromptWindow = true;
```

Comment:
same issues like for IE- see above comment

also i don't see here handling for HttpServerTimeOut so please add it